src/components/TaskList/TaskList.jsx

```diff
@@ -1,11 +1,37 @@
 import * as React from 'react';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import Stack from '@mui/material/Stack';
+import { styled } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
 import './TaskList.css';
 
+const Item = styled(Paper)(({ theme }) => ({
```

Comment:
Good use of the styled util from MUI, though I usually find it to be more readable and maintainable to create an entire new component for "Item" and just export it from there to be used throughout the app, you can use `useTheme()` from MUI to achieve the same dynamic behavior with the dark/light mode.